plugins/Makefile.mk

```diff
@@ -23,3 +23,8 @@ include plugins/dmx4linux/Makefile.mk
 include plugins/e131/Makefile.mk
 include plugins/uartdmx/Makefile.mk
 endif
+
+dist_noinst_SCRIPTS += plugins/convert_README_to_header.sh
+
+plugins/%/PluginDescription.h: plugins/%/README.md plugins/convert_README_to_header.sh plugins/Makefile.mk
```

Comment:
Currently, the `PluginDescription.h`s are also not built if the flag is false, since it only gets added to `BUILT_SOURCES` if the flag is set.

Your solution would definitely be easier to setup, I do just wonder if it's also easy to maintain.